apiserver/facades/client/spaces/spaces_test.go

```diff
@@ -1114,63 +1102,6 @@ func (s *LegacySuite) TestCreateSpacesBlocked(c *gc.C) {
 	c.Assert(err, jc.Satisfies, params.IsCodeOperationBlocked)
 }
 
-func (s *LegacySuite) TestSupportsSpacesModelConfigError(c *gc.C) {
-	apiservertesting.SharedStub.SetErrors(
-		errors.New("boom"), // Backing.ModelConfig()
-	)
-
-	err := spaces.SupportsSpaces(&stubBacking{apiservertesting.BackingInstance})
-	c.Assert(err, gc.ErrorMatches, "getting environ: retrieving model config: boom")
-}
-
-func (s *LegacySuite) TestSupportsSpacesEnvironNewError(c *gc.C) {
-	apiservertesting.SharedStub.SetErrors(
-		nil,                // Backing.ModelConfig()
-		nil,                // Backing.CloudSpec()
-		errors.New("boom"), // environs.New()
-	)
-
-	err := spaces.SupportsSpaces(&stubBacking{apiservertesting.BackingInstance})
-	c.Assert(err, gc.ErrorMatches,
-		`getting environ: creating environ for model \"stub-zoned-networking-environ\" \(.*\): boom`)
-}
-
-func (s *LegacySuite) TestSupportsSpacesWithoutNetworking(c *gc.C) {
```

Comment:
This test was actually giving us something as per comments above. We are not handling the case yet where the provider does not implement the NetworkingEnviron interface. This test show us that in this case we should be returning not supported.

Reply:
sure, I can add this test